.circleci/config.yml

```diff
@@ -65,6 +65,13 @@ jobs:
           <<: *poetry_install
       - store_test_results:
           path: '.pytest_results'
+
+  test_package_javascript:
+    executor: noos-ci/default
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
       - run:
           name: JavaScript unit tests
           command: yarn install && yarn test
```

Comment:
🐼 

JS dependencies and Dash JS components already created and attached into the CI workspace (`build_library` job) ; `yarn test` should be enough for such a job (and quicker).

Reply:
I tried before with `yarn test` alone and it seemed the node modules were not cached and it failed. I can try again.

Reply:
Solved! Added `node_modules` folder to `persist_to_workspace` in the `build_library` job.
Cleaner and faster now! 👍